golem-worker-executor-base/tests/lib.rs

```diff
@@ -92,8 +91,6 @@ tag_suite!(guest_languages3, group7);
 tag_suite!(ts_rpc2, group8);
 tag_suite!(ts_rpc2_stubless, group8);
 
-tag_suite!(fork, group9);
```

Comment:
Let's not introduce a new group for this. The reason for these groups is only to run them in parallel on CI and they are grouped in a way to optimize the CI run time. You can add the new suite to `group3` for example.

Reply:
Sorry I see this was removed later :) Please tag the new suite as `group3`.

Reply:
Or there are no worker executor tests at all. Sorry got confused by trying to review commit by commit

Reply:
Yes, I had to go through a bit of round trip there, just to realize, what I really wanted is the way it is now